docs/developer-docs/latest/getting-started/introduction.md

```diff
@@ -6,38 +6,38 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/getting-started/intro
+Strapi is an open-source project (see [LICENSE](https://github.com/strapi/strapi/blob/master/LICENSE) file for more information). The core project as well as the documentation and any related tool can be found in the [Strapi](https://github.com/strapi) GitHub organization.
 
-As it goes hand in hand with the open-source ecosystem, Strapi is open to contributions. The Strapi team appreciates every contribution, be it a feature request, bug report, or pull request. The following GitHub repositories are open-source and contributions-friendly:
+As it goes hand in hand with the open-source ecosystem, Strapi is open to contributions. The Strapi team appreciates every contribution, be it a feature request, bug report, or pull request. The following GitHub repositories are open-source and contribution-friendly:
 
-- [`strapi/strapi`](https://github.com/strapi/strapi): main repository of Strapi, which contains the core of the project. You can find the admin panel, core plugins, plugin providers, and the whole code that runs your Strapi application. Please read the [`CONTRIBUTING.md`](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md) file to have more information about contributions to the main repository.
-- [`strapi/documentation`](https://github.com/strapi/documentation): contains the whole documentation of Strapi. Please read the [contribution guide](https://github.com/strapi/documentation/blob/main/CONTRIBUTING.md) to have more information about contributions to the Strapi documentation.
-- [`strapi/design-system`](https://github.com/strapi/design-system): is the design system that is used in the admin panel. It brings consistency between the different admin plugins.
-- [`strapi/strapi-docker`](https://github.com/strapi/strapi-docker): contains the code used to generate the official Docker images for Strapi (available through our [Docker Hub](https://hub.docker.com/r/strapi/strapi)).
-- [`strapi/awesome-strapi`](https://github.com/strapi/awesome-strapi): contains everything the community built and all managed plugins. It is used as a central place to find and submit new packages such as plugins, middlewares, hooks, and general enhancements to the core of Strapi.
+- [`strapi/strapi`](https://github.com/strapi/strapi): the main repository of Strapi, which contains the core of the project. You can find the admin panel, core plugins, plugin providers, and the complete code that runs your Strapi application. Please read the [Contribute to Strapi](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md) guide for more information about contributing to the main repository.
+- [`strapi/documentation`](https://github.com/strapi/documentation): the complete documentation of Strapi. Please read the [Contribute to the Strapi documentation](https://github.com/strapi/documentation/blob/main/CONTRIBUTING.md) guide for more information about contributing to the Strapi documentation.
```

Comment:
```suggestion
- [`strapi/documentation`](https://github.com/strapi/documentation): the complete documentation of Strapi. Please read the [contributing](https://github.com/strapi/documentation/blob/main/CONTRIBUTING.md) guide for more information about contributing to the Strapi documentation.

Reply:
Having "contributing [...] contributing" twice in one sentence is just bad style (in every language). (And why is "Please read the [Contribute to Strapi]" not reverted?)

Reply:
Personally I'd rather keep the file name to be honest since it's referencing a specific file in each GitHub repo. But other than that, as a native english speaker, the sentence structure in my suggestion is correct. It has more flow then the break your original change suggested.